packages/paste-website/src/data/word-list.json

```diff
@@ -64,6 +64,7 @@
   "quit": "Avoid. Unique to Mac. Use 'close' or 'cancel', depending on the situation.",
   "refresh": "Use as a call to action to ask the user to reload the page.",
   "remove": "Use when the item is still available and/or the user can undo the action.",
+  "revert": "Use when you are rolling back to an older version (likely because something broke after you updated). For example, 'Revert to an earlier version of Flex' or 'You can revert to an earlier version at any time.'",
```

Comment:
```suggestion
  "revert": "Use when you are rolling back to an older version (likely because something broke after you upgraded). For example, 'Revert to an earlier version of Flex' or 'You can revert to an earlier version at any time.'",
```
non-blocking